creator-node/src/services/sync/primarySyncFromSecondary.js

```diff
@@ -225,7 +270,11 @@ async function _primarySyncFromSecondary({
   } catch (e) {
     tracing.recordException(e)
     await SyncHistoryAggregator.recordSyncFail(wallet)
```

Comment:
yeah this call is now broken since every earlier step will return before coming here, maybe that is fine tho

Reply:
will refactor for both p2s and s2p flow

Reply:
will refactor this flow in this pr. will do followup pr for secondary sync from primary